### PR TITLE
blog post for April community call

### PR DESCRIPTION
**Pull request by k8hertweck**
Does this look ok, @rgaiacs? Also check out the [survey](https://docs.google.com/forms/d/e/1FAIpQLSf_ka_wAkDls1h5eDoActM4k_ohvhkx46rHxYU00FfE-UgvOQ/viewform?usp=sf_link). Anything else we want to ask?

@tracykteal, anything else you'd like to include in the survey as a follow up to your previous questions?

### CODE REVIEW

File: _posts/2017/04/2017-04-14-installer-community-call.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+editor." While a choice of software may seem trivial, this particular discussion has led 
+to more than one heated exchange among programmers. In fact, the [Discuss list in March] 
+(http://lists.software-carpentry.org/pipermail/discuss/2017-March/thread.html) 
+was very active concerning issues with the Windows installer, which provides access to the 
+nano text editor through Git for Windows (used in both the Software Carpentry Shell and Git 
+lessons). That discussion eventually transitioned to consideration of alternative software 
+tools we might consider using and moved over to [GitHub](https://github.com/carpentries/conversations/issues/11), 
+where you can read more about the opinions and options voiced by participants.
+
+To help us understand how our community feels about the software we use for teaching the
```
Comment (tracykteal) [inline _posts/2017/04/2017-04-14-installer-community-call.md]: feels -> thinks

**Comment by tracykteal** [comment]
Survey looks good! Comment here and then a few suggestions.

I feel like the blog post and the survey go a little back and forth on 'software tools' generally versus 'software tools for teaching shell'. It seems like you want to be focusing on the latter, at least for the survey. For the community call, is that the focus of the discussion, or is it more a general 'what software should we teach' and 'how should we decide what software to teach'?

Survey recommendations, with the perspective that you're focusing on the shell tools:

In "What describes your role in the Software Carpentry community?" add, 'check all that apply'

"Which of the tools you've used in a workshop seemed to perform best? Which did you prefer?"-> "Which of the tools you've used in a workshop seemed to perform best for teaching shell? Which did you prefer?"

"Anything else you'd like for us to know about the topic of software installation?" -> "Anything else you'd like for us to know about the topic of software installation generally or for the shell specifically?"

**Comment by raynamharris** [comment]
What about Vim, Notepad, Emacs, and other text editors?

**Comment by tracykteal** [comment]
Good point. I've taught with both Vim and notepad.

**Comment by gvwilson** [comment]
Huh - in *my* day, we used `cat` for an editor.

**Comment by k8hertweck** [comment]
Thanks folks, I've made changes to both the survey and blog post as per your recommendations. In particular, I've tried to focus the conversation on the shell lesson (since I suspect we have a great deal to glean from those specific responses), but have fleshed out some content in the agenda for the meeting to save a bit of time to discuss larger issues. I also added Vim, Emacs, and Notepad to the survey as options. 

@gvwilson, were you editing files using `cat` while also walking uphill in the snow?

**Comment by k8hertweck** [comment]
In the interest of time, can you go ahead and merge this, @jduckles?

**Comment by rgaiacs** [comment]
> Does this look ok?

@k8hertweck Sorry for the delay. I went to holidays I left my laptop at home. The post looks great. Thanks for it.

**Pull request merged**